app/views/_init-count.html

```diff
@@ -0,0 +1,11 @@
+<!--
+  Expects the following variables:
+  podTemplate
+ -->
+
+<span ng-if="pod.spec.initContainers.length" class="text-muted small">
+  <ng-pluralize count="pod.spec.initContainers.length"
+                when="{'1': '&nbsp;Init container','other': '&nbsp;{} Init containers'}">
+  </ng-pluralize>
+  <a ng-href="{{pod | navigateToTabURL:'details'}}">View Details</a>
```

Comment:
`details` is not always the right tab depending on what kind of resource it is :/  Some resources need `configuration`

`pod` is not a pod, but a pod template. This link will only work for pods not other types like deployment configs.